src/inspector_js_api.cc

```diff
@@ -275,7 +275,21 @@ void Initialize(Local<Object> target, Local<Value> unused,
   Environment* env = Environment::GetCurrent(context);
 
   Agent* agent = env->inspector_agent();
-  env->SetMethod(target, "consoleCall", InspectorConsoleCall);
+
+  v8::Local<v8::Function> consoleCallFunc =
+      env->NewFunctionTemplate(InspectorConsoleCall, v8::Local<v8::Signature>(),
+                               v8::ConstructorBehavior::kThrow,
+                               v8::SideEffectType::kHasSideEffect)
+          ->GetFunction(context)
+          .ToLocalChecked();
+  // kInternalized strings are created in the old space.
+  const v8::NewStringType type = v8::NewStringType::kInternalized;
+  v8::Local<v8::String> name_string =
+      v8::String::NewFromUtf8(env->isolate(), "consoleCall", type)
+          .ToLocalChecked();
```

Comment:
This could probably use `FIXED_ONE_BYTE_STRING`

Reply:
maybe an env.h string?

Reply:
Added `FIXED_ONE_BYTE_STRING` for consistency with `conn_str` from `inspector::Initialize`